src/shared/components/date-bubble/index.js

```diff
@@ -1,4 +1,5 @@
 // Displays date bubble
+/* eslint-disable max-len */
```

Comment:
You can wrap a string over multiple lines  with either `+` or `\`. 

Also this is an expression which is easier to wrap. I think it should be built by a function rather than inline like this. Really hard to read between the template strings, use of `this.props`, and the ternary operator.